### PR TITLE
Removed blue border

### LINKED ISSUE #629: Removing blue border
image
Currently, there's a blue border around the sketch page which needs to be improved. Entire page should have the canvas

### PR DESCRIPTION
**Pull request by malavikarajeshvikraman**
Hi ,
I have tried to increase the size of canvas and remove blue borders (fix #629).Can you please review and if its alright merge it to the main repo.

### CODE REVIEW

File: src/pages/Sketch/components/Canvas.module.css [hunk 1]
```diff
@@ -1,8 +1,8 @@
 .canvas, .dark_canvas {
-    border: 2.5px solid blue;
     border-radius: 11px;
```
Comment (anushbhatia) [inline src/pages/Sketch/components/Canvas.module.css:2]: remove the space present here too

**Review by anushbhatia — CHANGES_REQUESTED** [review]

File: src/pages/Sketch/components/Canvas.module.css (hunk 1 above)
Comment (malavikarajeshvikraman) [reply src/pages/Sketch/components/Canvas.module.css:2]: Hey did you mean to remove border-radius attribute too ?

Comment (anushbhatia) [reply src/pages/Sketch/components/Canvas.module.css:2]: there's extra space as you have deleted just move that bit little up.

**Comment by anushbhatia** [comment]
just resolve the merge conflict will merge it.

**Comment by malavikarajeshvikraman** [comment]
Hi, I have merged it !

**Pull request merged**